File: src/ch/adesso/exam3/FizzBuzzIntegration.java
package ch.adesso.exam3;

import java.util.List;

public final class FizzBuzzIntegration {

    public static void main(String[] args) {
        List<Integer> number = RangeGenerator.generateRange();
        List<String> result = FizzBuzzConverter.convert(number);
        Printer.print(result);
    }
}

File: Exam3/Exam3/FizzBuzzIntegration.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Exam3
{
	public class FizzBuzzIntegration
	{
		private readonly int StartIndex = 1;
		private readonly int EndIndex = 100;

		public void FizzBuzzIntegrationHandler()
		{
			var rangeGenerator = new RangeGenerator();
			var fizzBuzzConverter = new FizzBuzzConverter();
			var printer = new Printer();

			var numbers = rangeGenerator.GenerateRange(StartIndex, EndIndex);
			var convertedNumbers = fizzBuzzConverter.ConvertNumber(numbers);
			_ = printer.PrintNumbers(convertedNumbers);
		}
	}
}
